packages/node/package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "build": "rollup -c",
-    "test": "jest --setupFiles dotenv-extended/config --runInBand --bail --forceExit",
+    "test": "jest --setupFiles dotenv-extended/config --runInBand --bail --forceExit --verbose",
```

Comment:
@cf19drofxots there was a reason you removed this no?

Reply:
It was an attempt to speed up the tests if no logs needed to be buffered & printed. Didn't seem to make a difference though.